### DISCUSSION TITLE
Allow disabling light/dark mode toggle

### DISCUSSION THREAD

**Discussion by Eric-Arellano** [pradyunsg/furo#650] [Ideas] [answered]
Our project is excited to add dark mode support thanks to Furo. It's a big reason we switched themes! 

But, we need to do some work first like choosing our color scheme & auditing all projects' images. We don't want to block switching to Furo on this.

So, we'd love the option in `conf.py` to disable the theme toggle button:

https://github.com/pradyunsg/furo/blob/e7f732ea5133d29fc6ca62af0455b16ffaab00dc/src/furo/theme/furo/page.html#L36-L43

https://github.com/pradyunsg/furo/blob/e7f732ea5133d29fc6ca62af0455b16ffaab00dc/src/furo/theme/furo/page.html#L76-L84

I'd be happy to add this if you'd like :)

**Comment by pradyunsg** [comment, marked as answer]
See https://github.com/pradyunsg/furo/issues/28
